__tests__/app.test.js

```diff
@@ -97,4 +96,24 @@ describe('5. get /api/users', () => {
             })
         })
     });
+});
+
+describe('6', () => {
+    test('returns 200 and updated article', () => {
+        return request(app)
+        .patch("/api/articles/1")
+        .send({ inc_votes : 55})
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article).toEqual({
+                article_id: 1,
+                title: 'Living in the shadow of a great man',
+                topic: 'mitch',
+                author: 'butter_bridge',
+                body: 'I find this existence challenging',
+                created_at: "2020-07-09T20:11:00.000Z",
+                votes: 155
+              })
+        })
+    });
```

Comment:
Great start with a happy path test, though you're missing a few tests you need to add for this endpoint, namely: 
-400 invalid article id (like an ID of 'banana')
-404 valid but non existent article id (like 99999999)
-missing or incorrect body- so something wrong with the inc votes body object